app/src/main/java/com/edwincobos/subscribersapp/subscriberslist/SubscribersListActivity.java

```diff
@@ -0,0 +1,39 @@
+package com.edwincobos.subscribersapp.subscriberslist;
+
+import android.os.Bundle;
+
+import com.edwincobos.subscribersapp.R;
+import com.edwincobos.subscribersapp.commons.abstracts.AbstractActivity;
+
+public class SubscribersListActivity extends AbstractActivity implements SubscribersListView {
```

Comment:
Sorry for the mistake, I thought the PR sent doesn't update from new commits. 
To the future, if there is a PR open I'm going to create a new branch for new commits